lib/markdown-preview-view.coffee

```diff
@@ -6,8 +6,10 @@ Grim = require 'grim'
 _ = require 'underscore-plus'
 fs = require 'fs-plus'
 {File} = require 'pathwatcher'
+String = require 'string'
```

Comment:
I'm not sure this is needed, strings are built-in and shouldn't require a third-party library.